stylesheets/_layout.tabs.scss

```diff
@@ -1,6 +1,7 @@
 .content-main {
     position: relative;
     display: table;
+    height: 100%;
```

Comment:
@jadu/pulsar ⚠️ Possibility of unintended consequences, had no effect in lexicon bit shout up if this concerns you

Reply:
Checked it in CMS and down to IE9. It seems to work properly without breaking anything.

Reply:
Tested in CXM all good.